packages/keeper-bots/src/converter-bot/queries/getVTokensFromUnderlying.ts

```diff
@@ -11,11 +11,17 @@ const getVTokensFromUnderlying = async (
   isolatedVTokens: IsolatedVTokensFromUnderlyingQuery["markets"];
 }> => {
   const isolatedPoolSubgraphClient = new SubgraphClient(getConfig().isolatedPoolsSubgraphUrl);
-  const corePoolSubgraphClient = new SubgraphClient(getConfig().corePoolSubgraphUrl);
   const { data: { markets: isolatedVTokens = [] } = { isolatedVTokens: [] } } =
     await isolatedPoolSubgraphClient.getIsolatedVTokensFromUnderlying(underlyingAddress);
-  const { data: { markets: coreVTokens = [] } = { coreVTokens: [] } } =
-    await corePoolSubgraphClient.getCoreVTokensFromUnderlying(underlyingAddress);
+  const corePoolSubgraphUrl = getConfig().corePoolSubgraphUrl;
+  let coreVTokens: CoreVTokensFromUnderlyingQuery["markets"] = [];
+  if (corePoolSubgraphUrl) {
+    const corePoolSubgraphClient = new SubgraphClient(getConfig().corePoolSubgraphUrl);
+    const { data: { markets } = { markets: [] } } = await corePoolSubgraphClient.getCoreVTokensFromUnderlying(
+      underlyingAddress,
+    );
+    coreVTokens = markets;
+  }
```

Comment:
nit: looks like this could be simpler, `corePoolSubgraphUrl` can be used in line 19 instead of getting it again from config and the client could return an empty array directly if there is no `corePoolSubgraphUrl`